content/en/docs/reference/controllers/admission-pod-preset.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Pod preset controller
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+
+The Pod preset admission controller injects configuration data into
```

Comment:
- Is the controller named PodPreset or is this just the name of the admission controller flag?
- Should Admission Controller be capitalized, throughout the pages?
- Capitalize Pod/Pods?
- Since the content, in lines 21 - EOF, is taken from `https://kubernetes.io/docs/concepts/workloads/pods/podpreset/` is your intention to remove the controller-related content from the original doc, leaving the information about the resource?
- Capitalize VolumeMount?

Reply:
Thanks for the feedback.

I'm not sure that controllers have official names. I suppose there's the name of the Go package (eg `expand` or the struct (eg `expandController`).
Is it useful to mention that in the docs?

Reply:
Hi @sftim. I believe there are established names for some of the controllers. It would be great to confirm the accepted names for each of the controllers. Checking the comments section for each controller _might_ help confirm the names and provide content, such as:  

`// ReplicaSetController is responsible for synchronizing ReplicaSet objects stored
// in the system with actual running pods.
type ReplicaSetController struct {`

`// TokenCleaner is a controller that deletes expired tokens
type TokenCleaner struct {`

Reply:
https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/#options lists controller names (in the `--controllers` parameter) that are visible to cluster operators.

Now I've seen that, I'm minded to put those lower-case names somewhere in the page for each controller.